_docs/ops/secrets.md

```diff
@@ -66,11 +66,18 @@ aws-vault add cloud-gov-govcloud
 
 ### Configure MFA for aws-vault
 
-All operators must have MFA enabled, which is taken care of as a part of the platform operator onboarding process.  This can be confirmed by checking in the AWS console under `Services -> IAM -> Users -> firstname.lastname -> Security Credentials`, or in the command line with `aws iam list-virtual-mfa-devices`.  Confirm that MFA is enabled for your account, then use the following commands to add the ARN of the MFA device to the local Amazon configuration in order to enable short lived tokens:
+All operators must have MFA enabled, which is taken care of as a part of the platform operator onboarding process.  This can be confirmed by checking in the AWS console under `Services -> IAM -> Users -> firstname.lastname -> Security Credentials`.
+
+After confirming that MFA is enabled for your account, run these commands to set command line variables (where `mfa_serial` is the full ARN of your MFA device found in the AWS console):
+
+```sh
+me=firstname.lastname
```

Comment:
I don't see where `me` gets used - am I missing it?

Reply:
Nope. I see that got removed with the docs rewrite.

I could interpolate it into the `mfa_serial`, but that seems like an unnecessary step. I'll probably just remove it altogether